src/Button.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+const Button = ({ handleOnClick }) => {
+  return (
+    <div>
+      <button onClick={handleOnClick}>Get A user</button>
```

Comment:
If a component only returns a native html element (with no additional styling or functionality and passes a function to its parent, it makes more sense to just use the element in the parent component directly.